src/app/core/services/authentication.service.ts

```diff
@@ -57,6 +58,34 @@ export class AuthenticationService {
     return from(this.auth.authState);
   }
 
+  updateDisplayName(name: string) {
+    this.auth.authState.pipe(take(1)).subscribe((user) => {
+      if (user) user.updateProfile({ displayName: name });
+    });
```

Comment:
I think I wrote that before. Even though I repeat it it's an opinion only and I only want to make sure you're aware of it 😁

I always prefer an empty subscribe `stream.pipe(tap(() => {//side effect}).subscribe()`. That way streams are more portable. The subscribe is not always part of a subscribe. Even more if you collect the stream in some place but another place uses it.